src/main/g8/README.md

```diff
@@ -14,15 +13,26 @@ The intention is to use `sbt new` on this library's exterior g8 template, then t
 - `sbt-native-packager` plugin for **building a linked binary** _(builds ahead-of-time using GraalVM statically linking and building all dependencies in a docker image via `sbt graalvm-native-image:packageBin`)_
 - `scalatest` for **testing** _(default test runner via `sbt test`)_
 - `sbt-scoverage` for **test coverage** _(statement-level coverage via `sbt coverage`)_
-- `scalalogging -> slf4j -> log4j2` for **logging** _(scala code -> interface -> mechanism)_
+- `scalalogging -> logback-classic` for **logging** _(`logback-classic` is used as a backend, but it is very easy to [replace this with something like log4j2](https://github.com/sbchapin/quickstart-scala-sbt.g8/blob/master/src/main/g8/build.sbt#L19))_
```

Comment:
I would advise against the log4j2 part of this comment due to it adding reflective overhead which GraalVM native-image will take umbrage with (and consequently require additional configuration).  It's a good suggestion, but one I would bring in along with instruction and [explicit reflective configuration](https://github.com/oracle/graal/blob/master/substratevm/REFLECTION.md) (or alternatively wait for more SubstrateVM support).

Reply:
👍 I'll do that, if for nothing else just to keep the verbosity down. If there was a nice way of alluding to your explanation above I'd love a link to it. Maybe a fun facts section? I know for a fact a few people will be trying to slap log4j2 on this bad boy and it'd be nice to save them the trouble.

Reply:
I thought about this a tiny bit while I was touching up other parts of the readme. What do you think about this? https://github.com/sbchapin/quickstart-scala-sbt-native-image.g8/pull/1/files#diff-e6b572c427b2150bec2d15df72b1990bR218

Reply:
I like it!

Good callout that they gonna be trying, and I think we should get in front of that in `build.sbt` as well - Lord knows they ain't gonna read the readme.

Reply:
Lol, too true. I'll drop a quick line in the `build.sbt` 👍